packages/web/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Box, HStack, Spacer } from '@chakra-ui/react';
+import { NavLink } from './NavLink';
+import { NavProfileMenu } from './NavProfileMenu';
+
+export const NavBar: React.FC = () => {
+  return (
+    <>
+      <Box className="navbar" px={4}>
+        <HStack h={12} spacing={8} alignItems={'center'}>
+          <NavLink label="Home" url="/app" />
+          <NavLink label="Community" url="" />
+          <NavLink label="Videos" url="/app/videos" />
+          <Spacer />
+          <NavProfileMenu />
+        </HStack>
+      </Box>
+    </>
```

Comment:
- The fragment doesn't need to be there because there is only one element at the top level. The rest looks good, and then check out the main comment about the changes to the theme.
- I don't think we need the box here, we can put the styling on the `HStack` directly
- Instead of setting a static height for the `HStack`, put a vertical padding on the `NavLink` component to space it out more. Helps in case there are any overflows

```suggestion
    <Box className="navbar" px={4}>
      <HStack h={12} spacing={8} alignItems="center">
        <NavLink label="Home" url="/app" />
        <NavLink label="Community" url="" />
        <NavLink label="Videos" url="/app/videos" />
        <Spacer />
        <NavProfileMenu />
      </HStack>
    </Box>
```